pkg/cmd/cli/cmd/deploy.go

```diff
@@ -222,7 +222,7 @@ func (o DeployOptions) deploy(config *deployapi.DeploymentConfig, out io.Writer)
 	}
 
 	config.Status.LatestVersion++
-	_, err = o.osClient.DeploymentConfigs(config.Namespace).Update(config)
+	_, err = o.osClient.DeploymentConfigs(config.Namespace).UpdateStatus(config)
```

Comment:
I don't want a version of the CLI to exist which makes a call to the `status` subresource in order to trigger a deployment